index.js

```diff
@@ -42,13 +42,15 @@ function networkIdleCallback(fn, options = { timeout: 0 }) {
 
   messageChannel.port1.addEventListener('message', handleMessage);
   messageChannel.port1.start();
+  
+  return timeoutId;
 }
 
 function cancelNetworkIdleCallback(callbackId) {
   clearTimeout(callbackId)
 
   networkIdleCallback.__callbacks__ = networkIdleCallback.__callbacks__
-    .find(cb => cb.id === callbackId)
+    .filter(cb => cb.id === callbackId)
```

Comment:
This should ideally have been —
```suggestion
    .filter(cb => cb.id !== callbackId)
```

Reply:
Oh yeah, of course, thanks for the correction